app/Console/Commands/ExifLens.php

```diff
@@ -76,31 +76,59 @@ public function handle(): int
 						$photo->size_variants->getOriginal()->filesize = $localFile->getFilesize();
 						$updated = true;
 					}
-					if (empty($photo->iso) && !empty($info->iso)) {
+					if (
+						($photo->iso === null || $photo->iso === '') &&
+						$info->iso !== null &&
+						$info->iso !== ''
+					) {
```

Comment:
Places like this make me wonder if we really need to test for both `null` and empty string everywhere, but I realize that it may not be worth it to figure it out in every single spot...

Reply:
I am convinced that a comparison for `null` should suffice, because we convert all empty strings to `null` anyway in the middleware. But I deliberately did it this way, because I thought that someone might ask the exact opposite question if I had done it the other way around.